### ISSUE TITLE
Possible bug in Touch File Plugin

### ISSUE THREAD

**Issue by erviem99** [MvvmCross/MvvmCross#901]
I'm still using version 3.2.2 where I noticed the following which I believe is a bug under iOS 8. The same code is also present in version 3.5.

```
namespace Cirrious.MvvmCross.Plugins.File.Touch
{
    public class MvxTouchFileStore : MvxFileStore
    {
        public const string ResScheme = "res:";

        protected override string FullPath(string path)
        {
            if (path.StartsWith(ResScheme))
                return path.Substring(ResScheme.Length);

            return Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.MyDocuments), path);
        }
    }
}
```

In iOS 8 the use of `Environment.SpecialFolder`is not allowed.

Please see http://developer.xamarin.com/guides/ios/application_fundamentals/working_with_the_file_system/

**Comment by slodge** [comment]
I believe this is already resolved by Xamarin - https://github.com/MvvmCross/MvvmCross/issues/843

**Comment by erviem99** [comment]
Yes, it looks like it has been fixed in version 8.4.0.27.

I use version 8.4.0.15 though so I have to work around these issues myself...
